src/System.Security.Cryptography.Xml/tests/XmlDecryptionTransformTest.cs

```diff
@@ -134,6 +134,7 @@ public void LoadInnerXml_XmlValidUriForExcept()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "https://github.com/dotnet/corefx/issues/18690")]
```

Comment:
I believe Xunit creates instance of the class per test case - static variables might be problematic though - will check if there any - I don't want to add any refactoring in this PR